mac

```diff
@@ -142,8 +142,8 @@ if brew_is_installed 'brew-cask'; then
   brew untap caskroom/versions
 fi
 
-fancy_echo "Updating Homebrew formulas ..."
-brew update
+fancy_echo "Updating Homebrew..."
+cd "$(brew --repo)" && git fetch && git reset --hard origin/master && brew update
```

Comment:
@monfresh is `brew --repo` synonymous for `brew --repository` or something else?  I don't see it listed as an option/flag in the man page and wanted to double check. This is just looking for the user's installed Homebrew location, right?

Also, once [the PR you have open with Homebrew](https://github.com/18F/laptop/pull/132), will this be needed or is it okay to just leave as is regardless?